cms/static/js/views/assets.js

```diff
@@ -74,7 +82,7 @@ define(["jquery", "underscore", "gettext", "js/models/asset", "js/views/paging",
             renderPageItems: function() {
                 var self = this,
                 assets = this.collection,
-                hasAssets = assets.length > 0,
+                hasAssets = this.collection.assetType != '' || assets.length > 0,
```

Comment:
This needs to be ```!=```. Are you using JSHint with the .jshintrc file in the root of edx-platform? That provides a lot of good feedback to make sure that JS best practices are followed.